GameOfLife/validator.cpp

```diff
@@ -0,0 +1,51 @@
+#include "validator.h"
+
+Validator::Validator()
+{
+
+}
+
+void Validator::CouterOfFirstLineWidth(std::__cxx11::string readString, int &lineWidth)
+{
+    for(unsigned numberOfSymbolFromReadString = 0; numberOfSymbolFromReadString< readString.size();numberOfSymbolFromReadString++)
+    {
+        lineWidth++;
+        if(readString[numberOfSymbolFromReadString] == '\n' ) break;
+    }
+}
+
+Size Validator::getSizeOfBoard() const
+{
+    return *sizeOfBoard;
+}
```

Comment:
In case of not being successful Validator return false and fill string errorMessage with information about number of line where row differs from the rest.